src/main/charts/bitbucket/Changelog.md

```diff
@@ -0,0 +1,61 @@
+# Change Log
+
+## 0.11.0 
```

Comment:
Is this release meant to include the changes Yifei made (helm output for service URL) and the documentation updates? If so they arent included in the the release notes

Reply:
They were there but I have clarified it a bit:

```
* DCKUBE-308: Print service URL after installing helm chart (#157)`
```